test/membrane_element_audiometer/peakmeter_test.exs

```diff
@@ -0,0 +1,30 @@
+defmodule Membrane.Audiometer.PeakmeterTest do
+  use ExUnit.Case, async: true
+  import Membrane.Testing.Assertions
+  alias Membrane.Testing
+
+  @module Membrane.Audiometer.Peakmeter
+
+  test "integration" do
+    data = [1, 2, 3, 2, 1] |> Enum.map(&<<&1>>)
+
+    {:ok, pipeline} =
+      %Testing.Pipeline.Options{
+        elements: [source: %Testing.Source{output: data}, peakmeter: @module, sink: Testing.Sink]
+      }
+      |> Testing.Pipeline.start_link()
+
+    Testing.Pipeline.play(pipeline)
+    assert_pipeline_playback_changed(pipeline, _prev_state, :playing)
+
+    assert_pipeline_notified(pipeline, :peakmeter, :underrun)
+
+    Enum.each(data, fn payload ->
+      assert_sink_buffer(pipeline, :sink, %Membrane.Buffer{payload: received_payload})
+      assert payload == received_payload
+    end)
+
+    Testing.Pipeline.stop(pipeline)
+    assert_pipeline_playback_changed(pipeline, _, :stopped)
```

Comment:
Credo will complain about `_`. You don't have tu add it now, but you can run checks using config from here: https://github.com/membraneframework/membrane_rtp_plugin/blob/e32b6bc000db54dedda0ce7b08e999b24c6e51a1/.credo.exs

Reply:
Thanks, for the spot, I'll add it even though I don't have tu. I'll run the checks tu ;)